docs/getting-started/introduction.md

```diff
@@ -114,7 +114,7 @@ For **2D members**, there are several options on how to define LCS of [Structura
 
 ### LCS enum set on **"axis** x **or** y **by vector" \(2D members\)**
 
-When is LCS enum set on **"axis** x **or** y **by vector"** explanation is similar to [StructuralCurveMember](../structural-analysis-elements/structuralcurvemember.md)presented above with the following additional information:
+When is LCS enum set on **"axis** x **or** y **by vector"** explanation is similar to [StructuralCurveMember](../structural-analysis-elements/structuralcurvemember.md) presented above with the following additional information:
```

Comment:
This is unrelated, there was just missing space which I noticed while working on the nonlinear combination